modules/server/configTemplates/base.json

```
@@ -1,4 +1,4 @@
 {
-	"documentType": "",
-	"index": ""
+	"documentType": "file",
+	"index": "file_centric_1.0"
```

Please leave the `templates` empty. These will be used later on for automated config generation.

```suggestion
	"documentType": "",
	"index": ""
```